src/NuGet.Services.BasicSearch/Startup.cs

```diff
@@ -168,7 +168,11 @@ public void Configuration(IAppBuilder app)
             ServicePointManager.DefaultConnectionLimit = Int32.MaxValue;
             ServicePointManager.Expect100Continue = false;
             ServicePointManager.UseNagleAlgorithm = false;
-            
+
+            // Ensure that SSLv3 is disabled and that Tls v1.2 is enabled.
```

Comment:
Talked to Joel offline. Most of the `ServicePointManager` settings are inconsistent across the Gallery, Search Service, and jobs - the only consistent setting would be the `SecurityProtocol`. Splitting this into another repository would be overkill for now, so we will keep it as it is for now.